codalab/apps/web/models.py

```diff
@@ -1439,6 +1446,8 @@ def save(self, ignore_submission_limits=False, *args, **kwargs):
 
         # only at save on object creation should it be submitted
         if not self.pk:
+            if self.participant.user.username == 'chagrade_bot':
+                ignore_submission_limits = True
```

Comment:
wonder why this is showing up in here...

Reply:
:( not sure, but we certainly want that...